doc/changelog/02-specification-language/07825-rechable-from-evars.rst

```diff
@@ -0,0 +1,8 @@
+- In `refine`, new existential variables unified with existing ones are no
+  longer considered as fresh. The behavior of `simple refine` no longer depend on
+  the orientation of evar-evar unification problems, new existential variables
```

Comment:
```suggestion
  the orientation of evar-evar unification problems, and new existential variables
```

Also, is "orientation" the right word here?  I wasn't sure what was meant, but that may just be my lack of knowledge.

Reply:
I think "orientation" is correct; to me it means "left-to-right" vs "right-to-left", i.e., the difference between `unify x y` and `unify y x` when you need to bias one side over the other (e.g., if you're unifying two evars and need to decide which one gets filled with the other)

Reply:
"left-to-right" and "right-to-left" suggest "direction" as a useful wording.  Your choice.